README.md

```diff
@@ -100,38 +115,70 @@ Also, make sure your firewall is configured to allow inbound connections to port
 
 ### Binaries (Windows/Linux/macOS)
 
-Binary releases are provided for common operating systems and architectures:
+Binary releases are provided for common operating systems and architectures.
+The easiest method is to download Decrediton from the link below, which will include dcrd.
+Advanced users may prefer the Command-line app suite, which includes dcrd and dcrwallet.
 
 https://decred.org/downloads
 
+* How to verify binaries before installing: https://docs.decred.org/advanced/verifying-binaries/
+* How to install the CLI Suite: https://docs.decred.org/wallets/cli/cli-installation/
+* How to install Decrediton: https://docs.decred.org/wallets/decrediton/decrediton-setup/
+
 ### Build from source (all platforms)
 
-Building or updating from source requires the following build dependencies:
+<details><summary><b>Install Dependencies</b></summary>
 
 - **Go 1.13 or 1.14**
 
   Installation instructions can be found here: https://golang.org/doc/install.
-  It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
+  Ensure Go was installed properly and is a supported version:
+    ```sh
+    go version
```

Comment:
This should include a leading `$` to clearly indicate to the user it is something they type at the command line and is for *nix.

e.g.

```bash
$ go version
$ go env GOROOT GOPATH

Reply:
Fixed